### PR TITLE
Add a note about installing OpenSSL version 0.9.8 if deploying SAML on Ubuntu

### PR DESCRIPTION
**Pull request by jasonblais**
@enahum Please help with dev review?

### CODE REVIEW

File: source/deployment/sso-saml-adfs.md [hunk 1]
```diff
@@ -16,6 +16,8 @@ The following are basic requirements to use ADFS for Mattermost:
 
 2 - Before configuring SAML with Microsoft ADFS, make sure you have the [XML Security Library](https://www.aleksey.com/xmlsec/download.html)  installed on your Mattermost instance. The XML Security Library is usually included as part of Debian GNU/Linux.
 
+If you're deploying on Ubuntu, make sure you also have [OpenSSL version 0.9.8](https://www.openssl.org/source/) installed on your Mattermost instance.
```
Comment (enahum) [inline source/deployment/sso-saml-adfs.md]: OpenSSL version 0.9.8 or greater

**Comment by jasonblais** [comment]
Thanks, updated

**Comment by enahum** [comment]
@jasonblais one more thing, I believe that should be done on any linux distro, not only ubuntu

**Comment by enahum** [comment]
And Looking at the conversation with the customer they had to install `xmlsec1-openssl` in ubuntu, dunno if it applies to other distros

**Comment by jasonblais** [comment]
Closing to resubmit

**Pull request closed without merging**